src/Client.js

```diff
@@ -815,21 +818,64 @@ class Client extends EventEmitter {
             );
         }
 
+
+        const isBigFile = internalOptions.attachment?.data?.length > (1024 * 1024 * 79);
+
+        if (isBigFile) {
```

Comment:
why not `internalOptions.attachment?.data?.length > (1024 * 1024 * 79)`  ?

Reply:
Updated